client/reader/search-stream/style.scss

```diff
@@ -100,17 +100,33 @@ $reader-related-card-v2-breakpoint-small: "( max-width: 535px )";
 		}
 	}
 
+	.reader-related-card-v2__post {
+		max-height: 205px;
+
+		@media #{$reader-related-card-v2-breakpoint-small} {
+			max-height: 100px;
+		}
+
+		@media #{$reader-related-card-v2-breakpoint-medium} {
+			max-height: 100px;
+		}
+
+		.reader-related-card-v2__excerpt {
+			-webkit-line-clamp: initial;
```

Comment:
Added this in to override `-webkit-line-clamp: 3;` that's being set in the `post-excerpt` component.